velox/common/file/tests/FileTest.cpp

```diff
@@ -107,3 +108,51 @@ TEST(LocalFile, viaRegistry) {
   ASSERT_EQ(readFile->pread(0, 5, &buffer1), "snarf");
   lfs->remove(filename);
 }
+
+TEST(LocalFile, exist) {
+  filesystems::registerLocalFileSystem();
+  auto tempFile1 = ::exec::test::TempFilePath::create();
+  const auto& filename1 = tempFile1->path.c_str();
+  remove(filename1);
+  auto tempFile2 = ::exec::test::TempFilePath::create();
+  const auto& filename2 = tempFile2->path.c_str();
+  remove(filename2);
+  auto lfs = filesystems::getFileSystem(filename1, nullptr);
+  {
+    auto writeFile = lfs->openFileForWrite(filename1);
+    writeFile->append("snarf");
+    writeFile = lfs->openFileForWrite(filename2);
+    writeFile->append("snarf");
+  }
+  ASSERT_TRUE(lfs->exist(filename1));
+  ASSERT_TRUE(lfs->exist(filename2));
+  lfs->remove(filename1);
+  ASSERT_FALSE(lfs->exist(filename1));
+  ASSERT_TRUE(lfs->exist(filename2));
+  lfs->remove(filename2);
+  ASSERT_FALSE(lfs->exist(filename1));
+  ASSERT_FALSE(lfs->exist(filename2));
+}
+
+TEST(LocalFile, list) {
+  filesystems::registerLocalFileSystem();
+  auto tempFolder = ::exec::test::TempDirectoryPath::create();
+  const auto& filename1 = fmt::format("{}/1", tempFolder->path);
```

Comment:
It is not a best practice to name variables a1, a2, etc. Here, I suggest to name these a, b (without filename prefix).